src/js/smooch.jsx

```diff
@@ -232,4 +232,19 @@ export class Smooch {
     showSettingsNotification() {
         store.dispatch(showSettingsNotification());
     }
+
+    getConversation() {
+        const conversation = store.getState().conversation;
+        const convoNotFoundError = () => {
+            throw new Error('conversation not found');
+        };
+
+        if (conversation) {
```

Comment:
that condition will always be true, conversation is never undefined in the store.